tests/test_downloadermiddleware_httpcache.py

```diff
@@ -151,7 +151,7 @@ class FilesystemStorageGzipTest(FilesystemStorageTest):
 
     def _get_settings(self, **new_settings):
         new_settings.setdefault('HTTPCACHE_GZIP', True)
-        return super(FilesystemStorageTest, self)._get_settings(**new_settings)
+        return super()._get_settings(**new_settings)
```

Comment:
As you can see, originally, this `super()` was referring to the `FilesystemStorageTest` superclass instead of `FilesystemStorageGzipTest`, but it's the same (`FilesystemStorageTest` doesn't implement its own `._get_settings()` method) so I decided to keep it as simply `super()`.